app/src/main/res/values/strings.xml

```diff
@@ -258,6 +258,7 @@
     <string name="select_mode">Select Mode</string>
     <string name="add_beneficiary_option">Add Third Party Beneficiaries to your account.\nEither
         enter manually or scan account QR code</string>
+    <string name="invalid_qr">Please scan QRCode of other user Savings or Loan Account</string>
```

Comment:
So your message should be like this, `You can't make  action into your account, please scan other user Savings and Loans Account QRCode`